### PR TITLE
Multi orgs bug fixes

### PR DESCRIPTION
**Pull request by tristanhcole**
- bug fixes
- auto login invited users once password (and if required TFA token) are set

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Admin Sign Up Flow Fix](https://trello.com/c/GiE6eXKX/127-admin-sign-up-flow-fix)

File: app/client/api/authApi.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const registerAPI = ({body}) => {
     body: JSON.stringify(body)
     })
     .then(response => {
-      return handleResponse(response)
+      return response.json();
```
Comment (enjeyw) [inline app/client/api/authApi.js]: What was the issue that handle response was causing that requires non standard handling?

File: app/client/api/creditTransferAPI.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    var query_string = generateQueryString(query);
     if (query) {
-        const query_string = generateQueryString(query);
         var URL = `/api/credit_transfer/${query_string}`;
     } else if (path) {
-        URL = `/api/credit_transfer/${path}/`;
+        URL = `/api/credit_transfer/${path}/${query_string}`;
     } else {
-        URL = '/api/credit_transfer/';
+        URL = `/api/credit_transfer/${query_string}`;
```
Comment (enjeyw) [inline app/client/api/creditTransferAPI.js]: I used to be put off by string literals, now I lovvvve them 😂

File: app/client/api/authApi.js (hunk 1 above)
Comment (tristanhcole) [reply app/client/api/authApi.js]: TFA returns a 401 that causes `handleResponse()` to raise an error. This is the same for the login API (where TFA is also active). Just a special case :)

https://github.com/teamsempo/SempoBlockchain/blob/713de7fffc712b9a06052002bc70434f5fe7b542/app/client/utils.js#L54-L59

File: app/client/api/creditTransferAPI.js (hunk 2 above)
Comment (tristanhcole) [reply app/client/api/creditTransferAPI.js]: I feel like we could have a `generateFormattedURL(`/api/credit_transfer/`, query, path)` util that generates a correct URL automatically ...

**Comment by enjeyw** [comment]
I was literally going to suggest the same thing!

File: app/client/api/authApi.js (hunk 1 above)
Comment (enjeyw) [reply app/client/api/authApi.js]: Ohhh yeah that makes sense

File: app/client/api/creditTransferAPI.js (hunk 2 above)
Comment (enjeyw) [reply app/client/api/creditTransferAPI.js]: I was literally gonna suggest the same thing, but I couldn’t find any other location where we need to parse the path.

Comment (tristanhcole) [reply app/client/api/creditTransferAPI.js]: cool refactored

**Pull request merged**